pysat/_files.py

```diff
@@ -266,6 +268,81 @@ def __str__(self):
 
         return output_str
 
+    def __eq__(self, other):
+        """Perform equality check
+
+        Parameters
+        ----------
+        other : any
+            Other object to compare for equality
+
+        Returns
+        -------
+        bool
+            True if objects are identical, False if they are not
+
+        """
+        if not isinstance(other, self.__class__):
+            return False
+
+        checks = []
+        item_check = []
+        for item in self.__dict__:
+            item_check.append(item)
+            # Confirm each object has the same items
+            if item in other.__dict__:
+                # Define default comparison. Series comparisons can error
+                # for simply being different (files, _*_file_list).
+                # 'inst_info' contains a weakref back to Instrument that
+                # requires a different check.
```

Comment:
Yes, I know that.  Why is it important to say that in this comment?  Why is it not enough to simply use that test?

Reply:
I am surprised that a Series comparison would error rather than return False. I thought I was adding clarity but I can remove the comment.

Reply:
Well, you have other comments that say the same thing down below.  If you found this helpful, then other people might too.  But in that case I think you should reword it.  Probably specify `ValueError: Can only compare identically-labeled Series objects` exactly and say that this is raised instead of False.  But, since you also say that down below where you do the ValueError catch, I think it could be removed here.